specification/communication/data-plane/PhoneNumbers/preview/2023-10-01-preview/phonenumbers.json

```diff
@@ -640,36 +640,27 @@
       }
     },
     "/availablePhoneNumbers/reservations/{reservationId}": {
-      "put": {
+      "patch": {
         "tags": [
           "Browse, reserve and purchase available phone numbers"
         ],
-        "summary": "Initializes a new reservation with a given ID. By default the new reservation is empty, but it can optionally be created with an initial list of phone numbers.",
+        "summary": "Creates or updates a reservation by its ID.",
         "consumes": [
-          "application/json"
+          "application/merge-patch+json"
         ],
         "produces": [
           "application/json"
         ],
-        "operationId": "PhoneNumbers_CreateReservation",
+        "operationId": "PhoneNumbers_CreateOrUpdateReservation",
         "parameters": [
           {
             "$ref": "#/parameters/ApiVersionParameter"
           },
           {
-            "in": "path",
-            "name": "reservationId",
-            "description": "The id of the reservation.",
-            "required": true,
-            "type": "string"
+            "$ref": "#/parameters/ReservationIdParameter"
```

Comment:
The intent was to `$ref` these parameters everywhere used - not just in this one place; otherwise, there's not much point. This saves you typing, but also from definitions accidentally diverging, which is typically unintentional.

Reply:
I'm not sure I get this comment. For every API using the ReservationId or Reservation as parameters, they are now using the `$ref` instead of declaring them inline.

I didn't change any of the already existing APIs, though; is that what you are suggesting?

Reply:
Yes. There should be no breaking changes. It's more of a nit, though.